src/braket/aws/aws_quantum_job.py

```diff
@@ -134,6 +135,10 @@ def create(
                 to execute the job. Default: InstanceConfig(instanceType='ml.m5.large',
                 instanceCount=1, volumeSizeInGB=30).
 
+            distribution (str): A str that specifies how the job should be distributed. If set to
```

Comment:
Should we add a note that it's intended for use with >1 instance count?

Reply:
I don't think so. Data parallel distribution could also be used with a single multi-gpu instance

Reply:
Ah, I didn't realize that. In that case, is there any potential a user may want to create a local job with data parallel distribution if their local hardware supports it?

Reply:
I think we can cut local mode out of scope. According to this [link](https://sagemaker.readthedocs.io/en/stable/overview.html), SageMaker local mode does not support distributed training with local GPU